shell/TestPackage.ps1

```diff
@@ -0,0 +1,24 @@
+$ErrorActionPreference = "Stop"
+$LastExitCode = 0
+$env:PATH +=";$($env:SystemDrive)\\ProgramData\\chocolatey\\bin"
+
+# https://github.com/chocolatey/choco/issues/512
+$validExitCodes = @(0, 1605, 1614, 1641, 3010)
+
+$package  = ls c:\packages\*.nupkg
```

Comment:
Maybe you are testing other things. Who knows? I typically don't like changes that limit the possibility further. More thinking usually brings the best option available to meet all needs.